src/OpenSage.Game/Scripting/Actions/ActionLookup.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using ScriptAction = OpenSage.Data.Map.ScriptAction;
+using ScriptActionType = OpenSage.Data.Map.ScriptActionType;
+
+namespace OpenSage.Scripting.Actions
+{
+    public static class ActionLookup
+    {
+        // TODO?: This is technically unnecessary boilerplate; we could construct this table dynamically using reflection.
+        // If we want even more magic we could even figure out the parameters for each action and pre-parse them.
+        private static readonly Dictionary<ScriptActionType, ScriptingAction> Actions = new Dictionary<ScriptActionType, ScriptingAction>
+        {
+            // Misc
```

Comment:
The way these are grouped into files is pretty arbitrary, but it's probably the easiest compromise between everything in one file vs action per file.

I personally use GoTo anything for pretty much all code navigation, so I don't really have a strong opinion on organising these.